code/game/gamemodes/wizard/wizloadouts.dm

```diff
@@ -120,3 +120,36 @@
 	if(!user)
 		return
 	ADD_TRAIT(user, SM_HALLUCINATION_IMMUNE, MAGIC_TRAIT)
+
+/datum/spellbook_entry/loadout/fireball
+	name = "Fireball. Fireball. Fireball."
+	desc = "Who cares about the rest of the spells. Become an expert in fire magic. Devote yourself to the craft. The only spell you need anyways is <b>Fireball.</b><br>\
+		</i>Provides fire immunity, homing fireballs, rapid-fire fireballs, and some fireball wands. Provides no mobility spells. Replaces your robes with infernal versions.<i>"
```

Comment:
I think you mixed up the placements of the <i> and </i>

Reply:
Intended, in the spellbook, desc is surrounded with italics. `<i>[desc]</i>`